src/actions/render.js

```diff
@@ -140,7 +140,7 @@ const substitutionsForPlatform = platform => {
         { search: percentEncodingPattern, replace: "%%" },
         { search: "{{s}}", replace: "$@" },
         { search: "{{d}}", replace: "$d" },
-        { search: "\"", replace: "\\\"" }
+        { search: '"', replace: '\\"' }
```

Comment:
@LcTwisk This is what prettier suggest. I'm not sure what to think of it?

Reply:
Would prefer consistent quotes but I can live with this ;)